src/common/SPELLS/OTHERS.js

```diff
@@ -778,4 +778,15 @@ export default {
     name: 'Felshield',
     icon: 'ability-vehicle-shellshieldgenerator-green',
   },
+  //Aman'Thul's Vision
+  AMANTHULS_GRANDEUR: { //Pantheon Proc
+    id: 256832,
+    name: 'Aman\'Thul\'s Grandeur',
+    icon: 'ability_shaman_repulsiontotem',
+  },
+  GLIMPSE_OF_ENLIGHTENMENT: { //Regular Proc
+    id: 256818,
+    name: 'Glimpse of Enlightenment',
```

Comment:
These two `` //Regular Proc `` and ``//Pantheon Proc`` should be opposite right?

Reply:
http://www.wowhead.com/spell=256818/glimpse-of-enlightenment

http://www.wowhead.com/spell=256832/amanthuls-grandeur

I don't think so.

Reply:
done and done friend